File: Iniciante/URI 1002.java
// https://www.beecrowd.com.br/judge/pt/problems/view/1002

import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
	    double raio = scanner.nextDouble();
	    double pi = 3.14159;
	    double area = pi * (raio * raio);
	    System.out.println(String.format("A=%.4f", area));
    }
 
}

File: Iniciante/URI 1002.cpp
// https://www.beecrowd.com.br/judge/pt/problems/view/1002

#include <iostream>
#include <cmath>

using namespace std;
main(){
    double n, area, raio;
    cin >> raio;
    n = 3.14159;
    area = n * pow (raio, 2);
    cout << fixed; cout.precision(4);
    cout << "A="<< area << endl;
}